sdk/metric/metricdata/data.go

```diff
@@ -137,6 +137,70 @@ type HistogramDataPoint[N int64 | float64] struct {
 	Exemplars []Exemplar[N] `json:",omitempty"`
 }
 
+// ExponentialHistogram represents the histogram of all measurements of values from an instrument.
+type ExponentialHistogram[N int64 | float64] struct {
+	// DataPoints are the individual aggregated measurements with unique
+	// Attributes.
+	DataPoints []ExponentialHistogramDataPoint[N]
+	// Temporality describes if the aggregation is reported as the change from the
+	// last report time, or the cumulative changes since a fixed start time.
+	Temporality Temporality
+}
+
+func (ExponentialHistogram[N]) privateAggregation() {}
+
+// ExponentialHistogramDataPoint is a single histogram data point in a timeseries.
+type ExponentialHistogramDataPoint[N int64 | float64] struct {
+	// Attributes is the set of key value pairs that uniquely identify the
+	// timeseries.
+	Attributes attribute.Set
+	// StartTime is when the timeseries was started.
+	StartTime time.Time
+	// Time is the time when the timeseries was recorded.
+	Time time.Time
+
+	// Count is the number of updates this histogram has been calculated with.
+	Count uint64
+	// Min is the minimum value recorded. (optional)
+	Min Extrema[N]
+	// Max is the maximum value recorded. (optional)
+	Max Extrema[N]
+	// Sum is the sum of the values recorded.
+	Sum N
+
+	// Scale describes the resolution of the histogram. Boundaries are
+	// located at powers of the base, where:
+	//
+	//   base = (2^(2^-scale))
+	Scale int32
+	// ZeroCount is the count of values that are rounded to zero. This bucket
+	// stores values that cannot be expressed using the standard exponential formula.
+	ZeroCount uint64
+
+	// PositiveOffset is the positive bucket index of the first entry in the
+	// PositiveCounts slice.
+	PositiveOffset int32
+	// PositiveCounts is an slice where PositiveCounts[i] carries the count of
+	// the bucket at index (offset+i). PositiveCounts[i] is the count of values
+	// greater than base^(offset+i) and less than or equal to base^(offset+i+1).
+	PositiveCounts []uint64
```

Comment:
The OTLP groups these into their own type: https://github.com/open-telemetry/opentelemetry-proto-go/blob/main/otlp/metrics/v1/metrics.pb.go#L1731

Any reason to not do the same here?

Reply:
If there was some functionality that was needed around the buckets I would have, but because this is a DTO with no functions I collapsed where possible.

I could change this if that's what is desired.